src/explanations/pancakeswap.yaml

```diff
@@ -0,0 +1,113 @@
+
+    ### How Does Pancakeswap Work?
+
+    Pancakeswap is built on the Binance Smart Chain, which utilizes the same technology as the Binance Chain but includes additional features such as smart contracts. This allows Pancakeswap to offer a range of decentralized finance (DeFi) services, including yield farming, liquidity provision, and trading.
+
+    Users can access Pancakeswap through a web browser or through a compatible wallet such as Trust Wallet or MetaMask. Once connected, users can browse the available assets and trade them on the exchange. Pancakeswap uses an automated market maker (AMM) model, which means that users don't have to wait for their orders to be matched with other buyers or sellers. Instead, the AMM algorithm calculates the price of each asset based on the supply and demand of the market.
+
+    ### Binance Smart Chain
+
+
+    Binance Smart Chain (BSC) is a blockchain network developed by the cryptocurrency exchange Binance. It is compatible with the Ethereum Virtual Machine (EVM), which means that it can run smart contracts and decentralized applications (DApps) that are built on Ethereum.
+
+
+    BSC was designed to offer the benefits of Ethereum's smart contract functionality, while also providing faster transaction speeds and lower fees. It uses a proof-of-staked-authority (PoSA) consensus mechanism, which means that block validation is performed by a group of validators who have staked a certain amount of the Binance Coin (BNB) cryptocurrency.
+
+
+    BSC has its own native cryptocurrency, called Binance Coin (BNB), which is used to pay transaction fees on the network. It also supports a wide range of other cryptocurrency assets, including those built on Ethereum and other networks.
+
+    ### The Exchange
+
+
+    Pancakeswap allows users to buy, sell, and trade a wide range of cryptocurrency assets, including popular coins like Bitcoin and Ethereum, as well as lesser-known altcoins and NFTs. Users can view the available assets and their prices on the Pancakeswap exchange, and can place orders to buy or sell using the AMM algorithm.
+
+
+    One of the key features of Pancakeswap is its liquidity pools, which allow users to provide liquidity to the exchange in exchange for a share of the fees generated by the trades. This helps to ensure that there is always a healthy supply of assets available for trading and allows users to earn a passive income by providing liquidity.
+
+
+    ### Yield Farming
+
+
+    Yield farming on PancakeSwap is a way for users to earn the platform's native token, CAKE, by providing liquidity to the DEX's liquidity pools and staking liquidity provider (LP) tokens in yield farms. When a user provides liquidity to a liquidity pool, they receive LP tokens that represent their share of the pool. These LP tokens can then be staked in a yield farm to earn CAKE rewards.
```

Comment:
Explain how yield farms are different from syrup pools e.g.

`Unlike Syrup Pools, Farms require you to stake two tokens to get LP Tokens, which you then stake in the Farm to earn rewards. This lets you earn CAKE while still keeping a position in your other tokens!`